apps/expo-go/ios/Podfile.lock

```diff
@@ -1384,7 +1384,7 @@ PODS:
     - Yoga
   - react-native-slider (4.4.2):
     - React-Core
-  - react-native-webview (13.6.3):
+  - react-native-webview (13.8.4):
```

Comment:
Can you check why the dependencies are not listed here (but they are in bare-expo) and hashes are different?